demo/api.py

```diff
@@ -0,0 +1,22 @@
+from flask import Flask
+from flask import request
+from flask import render_template
+import env_spec
```

Comment:
Instead of moving `env_spec.py` into the `demo` directory, you can make it available for importing throughout your project by using the following command:

```
pipenv install --dev -e 
```

Reply:
After running this command, I will still have the import statement in the demo/api.py?

Reply:
Yep! And you will still be able to have `env_spec.py` in the root directory.